guidelines/VERSIONING.md

```diff
@@ -16,17 +21,37 @@ The maintainers will edit the `CHANGELOG.md` according to the guidelines on [Kee
 
 ## Versions
 
-* **Patch** and **Minor**
+- **Patch** and **Minor**
 
   No special action required.
 
-* **Major**
+- **Major**
 
-  When a method is removed, the major version number MUST be incremented, and there MUST be a patch or minor release of the previous major version that includes a deprecation warning.
+  When a method is removed, or its signature changes in breaking ways, the major version number MUST be incremented, and there MUST be a patch or minor release of the previous major version that includes a deprecation warning. Consensus among the maintainers should be reached before doing a new major version since it creates a maintenance burden.
 
 ## Maintaining multiple versions
 
-When multiple versions are supported, we duplicate the `master`-branch to maintain support and leave the ability to patch and publish the older supported version.
-e.g. Our library is at v4 (`master`), but v3 (`master-3.x`) is still supported.
+Previous major versions are supported with fixes for 12 months after the release of the next major version. Fixes are not automatically backported, only where necessary. Features are only added on the latest major version.
```

Comment:
What to do when someone wants to introduce a new feature **in** an older version? E.g. when the component is part of a large application that can't be upgraded easily. Should there be specific guidelines for this case?

Reply:
We can consider a new minor release on an older branch, but obviously no breaking changes are possible. Regardless the responsibility of backporting the feature lies with the app developer who needs it. I will add a note accordingly.